R/NielsonKuparuk.R

```diff
@@ -0,0 +1,278 @@
+##Neilson, 2017
+##Megan Nguyen
+##2018
+
+#Clean files
+IC_WeirQ <- read.csv("/home/mnguyen/Neilson/IC_WeirQ.csv", header = TRUE)
+IC_WeirQ <- IC_WeirQ[,-3]
+names(IC_WeirQ) <- c("Date_adt", "Discharge_cms")
+
+Kup2AQ <- read.csv("/home/mnguyen/Neilson/Kup2AQ.csv", header = TRUE)
+names(Kup2AQ)
+
+ICKUP_Q <- read.csv("/home/mnguyen/Neilson/ICKUP_Q.csv", header = TRUE)
+str(ICKUP_Q)
+results <- grepl(pattern = "^\\d", ICKUP_Q$Date_adt)
+delete <- c()
+for(i in 1:length(ICKUP_Q$Date_adt)){
+  if(results[i] == FALSE){
+    delete <- c(delete, i)
+  }
+}
+
+ICKUP_Q <- ICKUP_Q[-delete,]
+names(ICKUP_Q)
+
+Kup2TQ <- read.csv("/home/mnguyen/Neilson/Kup2TQ.csv", header = TRUE)
+
+Kup3BQ <- read.csv("/home/mnguyen/Neilson/Kup3BQ.csv", header = TRUE)
+
+Kup3TQ <- read.csv("/home/mnguyen/Neilson/Kup3TQ.csv", header = TRUE)
+
+Kup4Q <- read.csv("/home/mnguyen/Neilson/Kup4Q.csv", header = TRUE)
+
+Kup5Q <- read.csv("/home/mnguyen/Neilson/Kup5Q.csv", header = TRUE)
+names(Kup5Q) <- c("Date_adt", "Discharge_cms")
+
+Kup8Q <- read.csv("/home/mnguyen/Neilson/Kup8Q.csv", header = TRUE)
+names(Kup8Q) <- c("Date_adt", "Discharge_cms")
+
+Kup9Q <- read.csv("/home/mnguyen/Neilson/Kup9Q.csv", header = TRUE)
+results <- grepl(pattern = "^\\d", Kup9Q$Date_adt)
+delete <- c()
+for(i in 1:length(Kup9Q$Date_adt)){
+  if(results[i] == FALSE){
+    delete <- c(delete, i)
+  }
+}
+
+Kup9Q <- Kup9Q[-delete,]
+
+
+files <- list(IC_WeirQ, ICKUP_Q, Kup2AQ, Kup2TQ, Kup3BQ, Kup3TQ, Kup4Q, Kup5Q, Kup8Q, Kup9Q)
+
+#Separate date and time in order to edit
+
+for(i in 1:length(files)){
+  files[[i]] <- separate(data = files[[i]], col = Date_adt, into = c("Date", "Time"), sep = " ")
+}
+
+##Add ":00" at end of time
+for(i in 1:length(files)){
+  for(j in 1:length(files[[i]]$Time)){
+    files[[i]]$Time[j] <- paste(files[[i]]$Time[j], ":00", sep = "")
+  }
+}
+
+##Make hour two digits
+for(i in 1:length(files)){
+  for(j in 1:length(files[[i]]$Time)){
+    pattern = "^\\d{2}[[:punct:]]"
+    results <- grepl(pattern, files[[i]]$Time)
+    if(results[j] == FALSE){
+      files[[i]]$Time[j] <- paste("0", files[[i]]$Time[j], sep = "")
+    }
+  }
+}
+
+#Change date format
+files[[1]]$Date <- as.Date(files[[1]]$Date, format = "%m/%d/%Y")
+for(i in 1:length(files[[1]]$Date)){
+if(isTRUE(files[[1]]$Date[i] < as.Date("2000-01-01"))){
+  files[[1]]$Date[i] <- format(files[[1]]$Date[i], "20%y-%m-%d")
+}else{
+  files[[1]]$Date[i] <- format(files[[1]]$Date[i])
+}
+}
+
+for(i in 2:length(files)){
+    files[[i]]$Date <- as.Date(files[[i]]$Date, format = "%m/%d/%Y")
+  }
+
+#Combine date and time columns
+for(i in 1:length(files)){
+  files[[i]] <- unite(files[[i]], Date_adt, c(Date, Time), sep = " ")
+}
+
+#Format Date_adt
+for(i in 1:length(files)){
+  format(as.POSIXct(files[[i]]$Date_adt, tz = ""), format = "%Y-%m-%d %H:%M:%S")
+}
+
+#Data file editing complete!
+
+#Reassign edited files to new file names
```

Comment:
Tip: if you want to separate your code into sections, just add `-------` or `######` after your comments. more info: https://support.rstudio.com/hc/en-us/articles/200484568-Code-Folding-and-Sections